ansible/configs/osp-satellite-vm/instance_loop.yml

```diff
@@ -5,8 +5,8 @@
 # host-GUID.GUID.zone.com
 - loop:
     - "{{ _instance.metadata.hostname }}"
-    - "{{ _instance.metadata.hostname | regex_replace('-' ~ guid ~ '$', '.' + guid) }}"
-    - "{{ _instance.metadata.hostname | regex_replace('-' ~ guid ~ '$', '-' ~ guid ~ '.' + guid) }}"
+    #- "{{ _instance.metadata.hostname | regex_replace('-' ~ guid ~ '$', '.' + guid) }}"
+    #- "{{ _instance.metadata.hostname | regex_replace('-' ~ guid ~ '$', '-' ~ guid ~ '.' + guid) }}"
```

Comment:
Please remove lines. Do not comment out code, let version control track previous versions